app/jobs/application_job.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationJob < ActiveJob::Base
+  # Automatically retry jobs that encountered a deadlock
+  retry_on ActiveRecord::Deadlocked
+
+  # Most jobs are safe to ignore if the underlying records are no longer available
+  discard_on ActiveJob::DeserializationError
```

Comment:
This file is so terse, Rails is good at this stuff. I don't think the comments add much 🤔

Reply:
Fair 'nuf. They were boilerplate and I was feeling lazy. :)